src/pages/ProfilePage.tsx

```diff
@@ -0,0 +1,42 @@
+// import { useAuth } from "contexts/AuthContext";
+// import { useLogin } from "custom-hooks";
+
+import { auth } from "config/firebase-config";
+import { useUser } from "contexts";
+import { signOut } from "firebase/auth";
+
+export const ProfilePage = () => {
+  const {
+    userData: { user },
+  }: any = useUser();
+  // const { logoutHandler } = useLogin();
```

Comment:
commented code can be removed if not required.

Reply:
removed console.log